src/style.css

```
@@ -1 +1,16 @@
-/* styles go here */
+.fieldset {
+  margin-bottom: 20px;
+}
+
+label {
```

```suggestion
.label {
```
don't style tags, style classes. add every `<label>` `class="label"`